volentix/settings.py

```diff
@@ -334,3 +336,36 @@ def celery_base_data_hook(request, data):
     "http": f"http://{CRAWLERA_AUTH_TOKEN}:@{CRAWLERA_PROXY_HOST}:{CRAWLERA_PROXY_PORT}/"
 }
 # EOF crawlera settings
+
+LOGGING_CONFIG = None
+LOGDNA_TOKEN = config('LOGDNA_TOKEN', default='this-token-does-not-work')
+DEPLOYMENT_ENV = config('ENV', default='dev')
+
+# # LogDNA settings
+# LOGGING = {
+#     'version': 1,
+#     'disable_existing_loggers': False,
+#     'handlers': {
+#         'logdna': {
+#             'level': logging.DEBUG,
+#             'class': 'logging.handlers.LogDNAHandler',
+#             'key': config('LOGDNA_TOKEN', default='this-token-does-not-work'),
+#             'options': {
+#                 'app': 'venue',
+#                 'env': config('ENV', default='dev'),
+#                 'index_meta': True
+#             },
+#         },
+#     },
+#     'loggers': {
+#         '': {
+#             'handlers': ['logdna'],
+#         },
+#         'celery': {
+#             'handlers': ['logdna'],
+#             'level': 'DEBUG',
+#             'propagate': False
+#         }
+#     }
+# }
+# # EOF LogDNA settings
```

Comment:
Do not forget to add a blank line at the end of files.

Reply:
Those commented lines will have to be removed. But yes, I need to add a blank line at the end of the file.